### DISCUSSION TITLE
When will the extension cater for PHP?

### DISCUSSION THREAD

**Discussion by lexman1958** [prettier/prettier-vscode#2921] [General]
The Prettier extension is fantastic but i am hoping that it will also format PHP language. At the moment we are using too many extensions and would good if we could reduce the load on VS code.

**Comment by trymeouteh** [comment]
Would like to see PHP support as well. PHP is a popular web language.

**Reply by murugappanrm** [reply]
Looks like there is no reply for our request. Many of us would like PHP to be included.
